modules/_labs/color-picker/react/lib/parts/ColorReset.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+
+import {borderRadius, colors, spacing, type, inputColors} from '@workday/canvas-kit-react-core';
+
+import {Swatch} from './Swatch';
+
+export const ResetButton: React.FunctionComponent<Props> = ({id, onClick, resetColor, label}) => {
+  const handleResetColor = () => onClick(resetColor);
+
+  const handleKeyDown = (event: React.KeyboardEvent) => {
+    if (event.key === 'Enter') {
+      handleResetColor();
+    }
+  };
+
+  return (
+    <Container id={id} onClick={handleResetColor} onKeyDown={handleKeyDown}>
+      <Swatch color={resetColor} showCheck={false} />
+      <Label id="canvas-color-picker--reset-label">{label || 'Reset'}</Label>
+    </Container>
+  );
+};
+
+export interface Props {
+  id?: string;
+  label?: string;
+  resetColor: string;
+  onClick: Function;
```

Comment:
Hmm. I thought we had a lint rule that prevents use of `Function`. It isn't very type-safe.

From the use, it looks like the signature is:
```ts
onClick: (value: string) => void
```

But that is weird for an `onClick` signature, right? If this is a reset button, why not use a `<button type="reset">` and make the popup wrapped in a `<form>` element? This way, clicking this button just invokes `onReset` of the form. This avoids the odd type signature of `onClick` passing a string rather than a value.